### PR TITLE
added template for Microsoft OMS

### PR DESCRIPTION
**Pull request by krnese**

### CODE REVIEW

File: oms/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2Foms%2F%2Fazuredeploy.json) 
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Foms%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+Your time is precious, why spend it on trying to integrate multiple tools. Lose the product palooza and unify on a single solution, providing management at your fingertips. Get up and running in seconds and have real insights in just hours, giving you faster time to value.
+
+## Microsoft OMS Azure Resource Manager Template
+
+This template will deploy *all* the OMS services into a Resource Group
```
Comment (gatneil) [inline oms/README.md]: I assume the 'a' is meant to be an 'A'?

File: oms/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,1210 @@
...
+    "runbooks": {
+      "omsABUscript": {
+        "name": "AzureIaasBackup",
+        "version": "1.0.0.0",
+        "description": "Runbook to automatically backup IaaS VMs within the ABU region",
+        "type": "PowerShell",
+        "Id": ""
+      }
+    },
+    "_assetLocation": "https://raw.githubusercontent.com/krnese/AzureDeploy/master/OMS/MSOMS/AzureIaaSBackup",
```
Comment (gatneil) [inline oms/azuredeploy.json]: please see "nested templates" in the best practices guide: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#best-practices

File: oms/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,1210 @@
...
+        "name": "RS0",
+        "tier": "Standard"
+      },
+      "properties": { },
+      "resources": [
+        {
+          "apiVersion": "2016-06-01",
+          "name": "HyperVPrimarySite",
+          "type": "replicationFabrics",
+          "id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.RecoveryServices/vaults/', parameters('omsRecoveryVaultName'), '/replicationFabrics/', 'HyperVPrimarySite')]",
```
Comment (gatneil) [inline oms/azuredeploy.json]: is there a reason not to use the [resourceId function](https://azure.microsoft.com/en-us/documentation/articles/resource-group-template-functions/#resourceid)? Would make this simpler

File: oms/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,1210 @@
...
+          "properties": {
+            "friendlyName": "PrimarySite",
+            "fabricType": "HyperVSite"
+          }
+        },
+        {
+          "apiVersion": "2016-06-01",
+          "name": "DefaultVMwarePolicy",
+          "type": "replicationPolicies",
+          "id": "[concat('/subscriptions/', subscription().subscriptionId, '/resourceGroups/', resourceGroup().name, '/providers/Microsoft.RecoveryServices/vaults/', parameters('omsRecoveryVaultName'), '/replicationPolicies/', 'DefaultVMwarePolicy')]",
```
Comment (gatneil) [inline oms/azuredeploy.json]: resourceId

File: oms/azuredeploy.json [hunk 5]
```diff
@@ -0,0 +1,1210 @@
...
+      "location": "[parameters('omsRecoveryVaultRegion')]",
+      "name": "[variables('recoveryStorageAccountName')]",
+      "type": "Microsoft.Storage/storageAccounts",
+      "comments": "Storage account for ASR",
+      "properties": {
+        "accountType": "[variables('recoveryStorageType')]"
+      }
+    },
+    {
+      "apiVersion": "2015-11-01-preview",
```
Comment (gatneil) [inline oms/azuredeploy.json]: is there a non-preview version we can use?

File: oms/azuredeploy.json (hunk 2 above)
Comment (krnese) [inline oms/azuredeploy.json]: the template will be re-structured to use nested once we have implemented some changes in the RPs.

Comment (gatneil) [inline oms/azuredeploy.json]: The best practice still applies for other dependent resources like scripts :)

File: oms/README.md (hunk 1 above)
Comment (krnese) [inline oms/README.md]: where`? The text itself is from marketing, so assume it is correct.

File: oms/azuredeploy.json (hunk 2 above)
Comment (krnese) [inline oms/azuredeploy.json]: scripts will be re-located and more will be added prior to ignite.

File: oms/azuredeploy.json (hunk 5 above)
Comment (krnese) [inline oms/azuredeploy.json]: not at this time. this is to showcase some new capabilities which is only present in these preview versions.

File: oms/README.md (hunk 1 above)
Comment (gatneil) [inline oms/README.md]: 'aLL'. It looks like it is meant to be in all-caps, so probably 'ALL'

File: oms/azuredeploy.json (hunk 2 above)
Comment (gatneil) [inline oms/azuredeploy.json]: ok

File: oms/azuredeploy.json (hunk 5 above)
Comment (gatneil) [inline oms/azuredeploy.json]: ok

**Comment by gatneil** [comment]
Travis CI is having an issue; manually tested; merging

**Pull request merged**